docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md

```diff
@@ -0,0 +1,147 @@
+---
```

Comment:
@ManuelDittmar you need to include the guide in the sidebar to show on the website.
please add it after this line:
https://github.com/camunda/camunda-platform-docs/blob/0efe538eb634ce7e51559949c6a46bc635a8bd2b/sidebars.js#L655